kvui.py

```diff
@@ -770,6 +774,42 @@ def fix_heights(self):
             element.height = max_height
 
 
+class ApAsyncImage(AsyncImage):
+    def is_uri(self, filename: str) -> bool:
+        if filename.startswith("ap:"):
+            return True
+        else:
+            return super().is_uri(filename)
+
+
+class ImageLoaderPkgutil(ImageLoaderBase):
+    def load(self, filename: str) -> typing.List[ImageData]:
+        # take off the "ap:" prefix
+        module, path = filename[3:].split("/", 1)
+        data = pkgutil.get_data(module, path)
+        return self._bytes_to_data(data)
+
+    def _bytes_to_data(self, data: typing.Union[bytes, bytearray]) -> typing.List[ImageData]:
+        from PIL import Image as PImage
+        p_im = PImage.open(io.BytesIO(data)).convert("RGBA")
+        im_d = ImageData(p_im.size[0], p_im.size[1], p_im.mode.lower(), p_im.tobytes())
+        return [im_d]
+
+
+# grab the default loader method so we can override it but use it as a fallback
+DefaultLoad = ImageLoader.load
```

Comment:
variable name

- This isn't a type, so this isn't the right capitalization pattern for it.
- Is there any reason for people to import this from the module and use it directly? If not it should begin with underscore (`_`)

maybe `_original_load` or `_original_image_load` or `_original_image_loader_load`